CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * fix accessibility hint in multi-select chat list title
 * update translations, revise english source
 * update to core90
+* register on system for `dcaccount:` scheme
```

Comment:
this is wrong here, maybe just drop the entry, we write the changelog on releases currently.